cmd/tailscale/cli/ssh.go

```diff
@@ -115,8 +111,7 @@ func runSSH(ctx context.Context, args []string) error {
 		}
 
 		argv = append(argv,
-			"-o", fmt.Sprintf("ProxyCommand %q %s nc %%h %%p",
-				tailscaleBin,
+			"-o", fmt.Sprintf("ProxyCommand tailscale %s nc %%h %%p",
```

Comment:
This change is also a slight regression - `os.Executable` returns a full path to the binary, rather than just the bin name, adding a dependency on `PATH` pointing to the correct `tailscale` binary